### PR TITLE
Switch to oauth2 for fitbit

### LINKED ISSUE #10: fitbit doesn't support consumer key any more
Looks like we cannot get consumer key's anymore:

https://community.fitbit.com/t5/Web-API/OAuth-1-0a-removal-update/m-p/1272107#U1272107

### PR DESCRIPTION
**Pull request by ppetru**
This fixes #10 and makes fitsync work again.

### CODE REVIEW

File: auth_fitbit.py [hunk 1]
```diff
@@ -1,78 +1,109 @@
...
 
-    keys = gather_keys()
     credentials = dict(
-        client_key=CLIENT_KEY,
-        client_secret=CLIENT_SECRET,
-        resource_owner_key=keys['oauth_token'].encode('ascii'),
-        resource_owner_secret=keys['oauth_token_secret'].encode('ascii'))
+        client_id=client_id,
+        client_secret=client_secret,
+				access_token=server.oauth.token['access_token'],
```
Comment (tantalor) [inline auth_fitbit.py]: please fix indentation

File: auth_fitbit.py [hunk 2]
```diff
@@ -1,78 +1,109 @@
...
+				refresh_token=server.oauth.token['refresh_token'])
     yaml.dump(credentials, open('fitbit.yaml', 'w'))
+
+if __name__ == '__main__':
+	  main()
```
Comment (tantalor) [inline auth_fitbit.py]: please fix indentation

**Comment by ppetru** [comment]
Done, thanks for the quick review.

**Comment by tantalor** [comment]
Thanks for fixing this!

**Comment by everydaynerd** [comment]
Awesome - thanks!

**Comment by Csega** [comment]
Thank you!

**Pull request merged**